### DISCUSSION TITLE
Welcome to CounterForMessenger Discussions!

### DISCUSSION THREAD

**Discussion by Kubis10** [Kubis10/CounterForMessenger#37] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 21power21** [comment]
Hey, wanted to ask what am I missing. I downloaded files from fb, transfered other folders into inbox folder, then ran CFM.exe, selected inbox folder, but it seems nothing happened - it just counted the folders for me.
![image](https://github.com/Kubis10/CounterForMessenger/assets/164753606/34f9adac-cecc-4dd7-b28a-01e44beca32d)
What file should I focus on? It seems there is no file from which the program could load the values (in each convo only photos, gifs, audio and vids folders). Thank you in advance!

**Reply by Kubis10** [reply]
@21power21 check your config.txt in the same place as your CFM.exe, path should look like: `../messages/inbox/ ` and inbox folder should have only simple conversations folders

**Reply by 21power21** [reply]
The path appears to be right, I think there's a problem with the conversation folders. Each conversation folder contains other folders as you can see in the screenshot below
![image](https://github.com/Kubis10/CounterForMessenger/assets/164753606/b55d607b-d98a-4ddf-9527-50ab57fd5cd2)
Maybe the problem is that there is no "messages" folder in the particular convo folder, only media folders?

**Reply by Kubis10** [reply]
You should have files named message.json in this folder too

**Reply by 21power21** [reply]
If there's no file .json does it mean fb messed this up? I'm pretty sure I ticked .json format instead of .html when I requested for the files.

**Reply by Kubis10** [reply]
I just send new request to download my data from Facebook, so I will let you know if it is Facebook problem, but I'm pretty sure it is their problem or some kind of bug //edit folder should look like this: 
![image](https://github.com/Kubis10/CounterForMessenger/assets/50967586/917f6184-8dba-42dd-9733-3b27b123c2ae)

**Reply by 21power21** [reply]
I also tried to re-request so I'm curious what happens. Thanks for your effort man!

**Reply by 21power21** [reply]
I received new files and there's no .json file as well, maybe I'm doing something wrong when requesting..

**Reply by Kubis10** [reply]
Same result for me, so I think Facebook has some kind of problem because it is not normal

**Reply by 21power21** [reply]
So maybe I should try that once in a while and see if it's gonna work.. Thank you for now:)
